api/test/media_integration.py

```diff
@@ -105,6 +105,28 @@ def thumb(fixture):
     assert thumbnail_response.headers["Content-Type"].startswith("image/")
 
 
+def thumb_compression(fixture):
+    thumbnail_url = fixture["results"][0]["thumbnail"]
+
+    thumbnail_response = requests.get(thumbnail_url)
+    compressed_size = len(thumbnail_response.content)
+    thumbnail_response = requests.get(f"{thumbnail_url}?compressed=no")
+    actual_size = len(thumbnail_response.content)
+
+    assert compressed_size < actual_size
```

Comment:
Can you add a complimentary test for the `full_size` parameter?

Reply:
Added tests in 3c23121.